### PR TITLE
Added Update Employee

### PR DESCRIPTION
**Pull request by daiahnadine**

### CODE REVIEW

File: employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java [hunk 1]
```diff
@@ -1,4 +1,31 @@
 package com.employee.information.management.app.facade.employee.impl;
 
+import com.employee.information.management.app.model.Employee;
+import com.employee.information.management.data.employee.dao.EmployeeDao;
+import com.employee.information.management.data.employee.dao.impl.EmployeeDaoImpl;
+
 public class EmployeeFacadeImpl {
```
Comment (esdi-00) [inline employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java]: This does not implement EmployeeFacade

File: employee-information-management-system/src/main/java/com/employee/information/management/data/employee/dao/impl/EmployeeDaoImpl.java [hunk 2]
```diff
@@ -1,4 +1,80 @@
...
 
+import com.employee.information.management.app.model.Employee;
+import com.employee.information.management.data.connection.ConnectionHelper;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
 public class EmployeeDaoImpl {
```
Comment (esdi-00) [inline employee-information-management-system/src/main/java/com/employee/information/management/data/employee/dao/impl/EmployeeDaoImpl.java]: This does not implement EmployeeDao

**Review by esdi-00 — CHANGES_REQUESTED** [review]
see comments

File: employee-information-management-system/src/main/java/com/employee/information/management/data/employee/dao/impl/EmployeeDaoImpl.java (hunk 2 above)
Comment (daiahnadine) [reply employee-information-management-system/src/main/java/com/employee/information/management/data/employee/dao/impl/EmployeeDaoImpl.java]: Noted po, Sir. Thank you po.

File: employee-information-management-system/src/main/java/Main.java [hunk 3]
```diff
@@ -1,2 +1,129 @@
+import com.employee.information.management.app.facade.employee.EmployeeFacade;
+import com.employee.information.management.app.facade.employee.impl.EmployeeFacadeImpl;
+import com.employee.information.management.app.model.Employee;
+import com.employee.information.management.data.employee.dao.impl.EmployeeDaoImpl;
+
+import java.util.Scanner;
+
 public class Main {
+<<<<<<< Updated upstream
```
Comment (RoCS2024) [inline employee-information-management-system/src/main/java/Main.java]: why do you have stashed changes here?

File: employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/EmployeeFacade.java [hunk 4]
```diff
@@ -1,4 +1,18 @@
 package com.employee.information.management.app.facade.employee;
 
+import com.employee.information.management.app.model.Employee;
+
 public interface EmployeeFacade {
+<<<<<<< Updated upstream
```
Comment (RoCS2024) [inline employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/EmployeeFacade.java]: Another stashed changes here

File: employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java [hunk 5]
```diff
@@ -1,4 +1,48 @@
 package com.employee.information.management.app.facade.employee.impl;
 
+<<<<<<< Updated upstream
```
Comment (RoCS2024) [inline employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java]: Another one

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
Please clean the codes. Remove the stashes

File: employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java [hunk 6]
```diff
@@ -1,4 +1,41 @@
...
+public class EmployeeFacadeImpl implements EmployeeFacade {
+
+    //update employee
+    EmployeeDao employeeDao = (EmployeeDao) new EmployeeDaoImpl();
+
+    public EmployeeFacadeImpl(EmployeeDaoImpl employeeDao) {
+    }
+
+    @Override
+    public List<Employee> getAllEmployees() throws RuntimeException {
```
Comment (RoCS2024) [inline employee-information-management-system/src/main/java/com/employee/information/management/app/facade/employee/impl/EmployeeFacadeImpl.java:19]: why is this returning null?

File: employee-information-management-system/src/main/java/com/employee/information/management/data/employee/dao/impl/EmployeeDaoImpl.java [hunk 7]
```diff
@@ -1,4 +1,122 @@
...
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+public class EmployeeDaoImpl implements EmployeeDao {
+
+    @Override
+    public List<Employee> getAllEmployee() {
```
Comment (RoCS2024) [inline employee-information-management-system/src/main/java/com/employee/information/management/data/employee/dao/impl/EmployeeDaoImpl.java:17]: why does this return null?

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comments

**Pull request closed without merging**